apps/docs/content/components/Label.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Label
+status: Experimental
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=6084-28401'
+source: https://github.com/primer/brand/blob/main/packages/react/src/Label/Label.tsx
+storybook: '/brand/storybook/?path=/story/components-label--default'
+description: Use a Label to indicate the status of an object or to categorize it with metadata that describes it.
```

Comment:
Could we use a similar description to PCSS? 

```suggestion
Use Labels to add metadata or indicate status of items and navigational elements.
```

Reply:
I think the usage in Brand is a bit different as we don't put labels for navigational items, but headings/titles mainly.

What about something in between and simpler?

```suggestion
description: Use Labels to add metadata or indicate an object's status.
```